sdk/synapse/Azure.Analytics.Synapse.Artifacts/api/Azure.Analytics.Synapse.Artifacts.netstandard2.0.cs

```diff
@@ -1733,6 +1733,12 @@ public partial class AzureTableStorageLinkedService : Azure.Analytics.Synapse.Ar
         public Azure.Analytics.Synapse.Artifacts.Models.AzureKeyVaultSecretReference SasToken { get { throw null; } set { } }
         public object SasUri { get { throw null; } set { } }
     }
+    public partial class BigDataPoolParametrizationReference
+    {
+        public BigDataPoolParametrizationReference(Azure.Analytics.Synapse.Artifacts.Models.BigDataPoolReferenceType type, object referenceName) { }
+        public object ReferenceName { get { throw null; } set { } }
```

Comment:
We should avoid using `object` in public APIs.  Is it possible to give this a strong type prior to release?